README.md

```diff
@@ -2,6 +2,15 @@
 
 App that upgrades Starcounter's `/sys/` dependencies to Polymer 1.3
 
+## Purpose of this app
+
+Starcounter installer includes a bundle of commonly used client side libraries, all in particular version. In case you want to try out these libraries in a different version, your options are:
```

Comment:
Starcounter installer -> The Starcounter installer

used client side libraries, all in particular version -> used client-side libraries with set versions